src/main/java/com/configcat/TargetingRule.java

```diff
@@ -0,0 +1,52 @@
+package com.configcat;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Describes a targeting rule.
+ */
+public class TargetingRule {
+    /**
+     * The list of conditions that are combined with the AND logical operator.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.